docker-images/kafka-base/Dockerfile

```diff
@@ -1,41 +1,38 @@
-FROM centos:7
-
-ARG JAVA_VERSION
+FROM strimzi/java-base:latest
 ARG KAFKA_SHA512
 ARG KAFKA_VERSION
 
-RUN yum -y install java-${JAVA_VERSION}-openjdk-headless gettext nmap-ncat openssl && yum clean all -y
+RUN yum -y install gettext nmap-ncat && yum clean all -y
 
 # set Kafka home folder
 ENV KAFKA_HOME=/opt/kafka
 
-# Add kafka user with UID 1001
+# Add kafka user with UID 1002
 # The user is in the group 0 to have access to the mounted volumes and storage
-RUN useradd -r -m -u 1001 -g 0 kafka
+RUN useradd -r -m -u 1002 -g 0 kafka
```

Comment:
I think this change could cause problems with access rights to storage etc. Please keep the user as `1001`.